Creative/apps/creative-tv/src/pages/profile/[address]/[id]/index.tsx

```diff
@@ -10,5 +8,5 @@ export default function AssetDetails() {
   const setPermission = () => {
     //          <MediaRenderer src={`${asset?.storage?.ipfs?.url}`} />
   }
-  return <>Post: {router.query.id}</>
-}
\ No newline at end of file
+  return <p>Post: {router.query.id}</p>
```

Comment:
Why use `<p />` instead of `<Text />`?

Reply:
This was page is WIP. At the moment I didn't think it was important so I left for the `tabs` on the profile page.